app/design/frontend/UnitedArtsEducation/unitednow/web/css/source/_extend.less

```diff
@@ -14,3 +14,13 @@
         border-color: @active__color;
     }
 }
+
+.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__m) {
+    .fotorama{
+        //max height for product pages images, if it is not fullscreen
+        &:not(.fotorama--fullscreen) .fotorama__stage{
+            max-height:400px;
```

Comment:
Setting a fixed height is not a preferred way to handle this, particularly because at certain viewport sizes, it will have no effect, and at others, it will leave unnecessary whitespace.
If you look a little higher up in the DOM, you'll notice that `.product.media` and `.product-info-main` have percentage-based widths set that together add up to a little less than 100%, which has the effect of creating 2 sub-columns with some margin inside `.column.main`. I suggest that you approach this by decreasing the width of `.product.media` and increasing the width of `.product-info-main` to correspond, and the height of the image gallery will scale to match. This is really a rather subjective issue, but changing both columns as suggested should have the effect of brining more content above the fold.